Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLParameterScan.py

```diff
@@ -0,0 +1,233 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+from __future__ import (absolute_import, division, print_function)
+
+from mantid.api import *
+from mantid.kernel import *
+from mantid.simpleapi import *
+from SANSILLAutoProcess import needs_loading, needs_processing
+
+
+class SANSILLParameterScan(DataProcessorAlgorithm):
+    """
+    Performs treatment for scans along a parameter for D16.
+    """
+    progress = None
+    reduction_type = None
+    sample = None
+    absorber = None
+    container = None
+    sensitivity = None
+    default_mask = None
+    output = None
+    normalise = None
+    output2D = None
+    output_joined = None
+    observable = None
+    pixel_y_min = None
+    pixel_y_max = None
+
+    def category(self):
+        return 'ILL\\SANS;ILL\\Auto'
+
+    def summary(self):
+        return 'Integrate SANS scan data along a parameter'
+
+    def seeAlso(self):
+        return []
+
+    def name(self):
+        return 'SANSILLParameterScan'
+
+    def validateInputs(self):
+        issues = dict()
+        message = "Wrong number of {0} runs: {1}. Please provide none or exactly one."
+        sens_dim = self.getPropertyValue('SensitivityMap').count(',')
+        if not (self.getPropertyValue('OutputJoinedWorkspace') or self.getPropertyValue("OutputWorkspace")):
+            issues["OutputJoinedWorkspace"] = "Please provide either OutputJoinedWorkspace, OutputWorkspace or both."
+            issues["OutputWorkspace"] = "Please provide either OutputJoinedWorkspace, OutputWorkspace or both."
+        if sens_dim != 0:
+            issues['SensitivityMap'] = message.format("sensivity map", sens_dim)
+        if self.getPropertyValue('PixelYmin') > self.getPropertyValue("PixelYmax"):
+            issues["PixelYMin"] = "YMin needs to be lesser than YMax"
+            issues["PixelYMax"] = "YMax needs to be greater than YMin"
+        return issues
+
+    def setUp(self):
+        self.sample = self.getPropertyValue('SampleRuns')
+        self.absorber = self.getPropertyValue('AbsorberRuns').replace('+', ',')
```

Comment:
```suggestion
        self.absorber = self.getPropertyValue('AbsorberRuns').replace(',', '+')
```
The same in the next line.